benchmark/arrays/var-int.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
In our current tests, it is atypical to have a blank line after `use strict`. (In `test/parallel`, there are 943 tests that contain `use strict`. Of those, only 108 contain a blank line after `use strict`.) So I favored the prevailing convention of no blank line. (Although I see I was not quite 100% consistent with that here. Ugh.)

Anyway, I don't have any strong feelings about it, but that's why I did it the way I did.